server/lib/get-coverage.js

```diff
@@ -0,0 +1,46 @@
+import browserslist from 'browserslist'
+import { agents as caniuseAgents, region as caniuseRegion } from 'caniuse-lite'
+
+export function getTotalCoverage(browsers, region) {
+  let isGlobal = !region
+  let coverage = isGlobal
+    ? browserslist.coverage(browsers)
+    : browserslist.coverage(browsers, region)
+
+  // BUG `caniuse-db` returns coverage >100% https://github.com/Fyrd/caniuse/issues/6426
+  coverage = coverage > 100 ? 100 : coverage
+
+  return roundNumber(coverage)
+}
+
+export async function getBrowserVersionCoverage(id, version, region) {
+  let isGlobal = !region
+  if (isGlobal) {
+    return parseBrowserVersionCoverage(caniuseAgents[id].usage_global, version)
+  }
+
+  try {
+    if (region.includes('/')) {
```

Comment:
Let’s use allow-list RegExp instead of prohibited `/` symbol.

I am afraid that we can miss something and will get hacked.

Reply:
👍👍👍

It's great that you noticed it.
I think we'll be fine with something like this:

```js
let isValidRegionName = /^(alt-)?[A-Za-z]{2}$/.test(region);
```